src/main/resources/application.conf

```diff
@@ -12,65 +12,29 @@ kamon {
 
   show-aspectj-missing-warning = on
 
-  metric {
-    filters {
-      # Define filter for your actor names here
-      # Example 1: akka-actor.includes = [ "default/user/**" ]
-      # Example 2:
-      # akka-actor {
-      # includes = [ "my-app/user/job-manager", "my-app/user/worker-*" ]
-      # excludes = [ "my-app/system/**", "my-app/user/worker-helper" ]
-      # }
-      # Example 3:
-      akka-actor.includes = [ "default/user/testActor1", "default/user/testActor2" ]
-      akka-router.includes = [ "**" ]
-      akka-dispatcher.includes = [ "**" ]
-      trace.includes =  [ "**" ]
-      trace-segment.includes =  [ "**" ]
-      histogram.includes =  [ "**" ]
-      min-max-counter.includes =  [ "**" ]
-      gauge.includes =  [ "**" ]
-      counters.includes =  [ "**" ]
-      # For web projects
-      http-server.includes = [ "**" ]
-
-      counters.excludes =  [ "" ]
-      trace-segment.excludes =  [ "" ]
-      histogram.excludes =  [ "" ]
-      min-max-counter.excludes =  [ "" ]
-      gauge.excludes =  [ "" ]
-      http-server.excludes =  [ "" ]
-    }
-  }
+  reporters = ["kamon.spm.SPMReporter"]
 
   spm {
-    token = "[place-token-here]"
+    token = "4a1366c2-9b4a-4f91-a0d2-c83b2e13e9a9"
     #Proxy support will be added in 0.6.8 version
```

Comment:
I think I saw some references to proxy info in the code, so maybe this comment is out of date?
If so, I wonder if proxy-xxxx things below are up to date?

Reply:
We support proxy setup and the property names are correct (we also support setting proxy using different java system property  (`http.proxyHost`, etc)). Will remove the comment.